### ISSUE TITLE
Non-committee TEMs cant change event descriptions

### ISSUE THREAD

**Issue by TeefaAly** [backstage-technical-services/hub#134]
#### Summary

When members have been assigned as TEMs to events on the website, they can not change the description of the event. They can still change the other fields such as time, venue, production charge, etc. But TEMs are unable to change only the description for the event they were assigned. From what I can tell, this is for all event types (events, meetings, socials, hidden). Committee members are still able to change descriptions

#### Actual behaviour

What is happening that you think shouldn't happen?
          TEMs should be able to change the description.

Pictures paint a thousand words and really help us understand what's going on - include screenshots if possible
          Whenever the TEM changed the description and presses save, it does not actually save it. After pressing save and reloading the page it goes back to the old description unchanged

#### Expected behaviour

What do you expect the website should do?
         TEMs should be able to change descriptions

#### When did it happen?

This will let us see if there is any helpful information in the server logs
          The first time I was made aware of this issue was about a month ago (around mid November 2021)

#### Urgency

How quickly does this need to be resolved? Is this preventing you or anyone else from being able to do something you/they need to?
         Medium Urgency. Not super critical for events, but would be nice
Use one of: Low, Medium, High, Very High

Thank you so much :)

**Comment by bnjns** [comment]
Hi @TeefaAly ,

Can you confirm the behaviour you're seeing? Do you have a text box which you edit but it's not saved when you press the save button, or are you just not presented with a text box?

Could you also provide an example event you're trying to edit, and what you're trying to change the description to (please don't include any personal or sensitive information)?

**Comment by TeefaAly** [comment]
I cant really test it on my account since I am on committee, but I tested it with a non-committee person yesterday and made a test event and made him TEM, I was able to change the description as I have a committee role, but they werent able to change it, even though they are TEM of the event. 
We both see the text box, when I press save on my account, it actually saves (cause committee role), but when they press it from their account, it doesnt save. 
This occurs for all events on the website. 
For example, the TEM for the last BUSMS November Overture show was not able to change the description of his event, as they wanted to include some information about the technical support we would be providing
